nvim/lua/nvchad/custom/configs/autocmds.lua

```diff
@@ -0,0 +1,40 @@
+local augroup = vim.api.nvim_create_augroup
+local autocmd = vim.api.nvim_create_autocmd
+-- local cmd = vim.api.nvim_create_user_command
+-- local namespace = vim.api.nvim_create_namespace
+-- Auto resize panes when resizing nvim window
+-- autocmd("VimResized", {
+--   pattern = "*",
+--   command = "tabdo wincmd =",
+-- })
+--
+autocmd("BufEnter", {
+  desc = "Open Neo-Tree on startup with directory",
+  group = augroup("neotree_start", { clear = true }),
+  callback = function()
+    if package.loaded["neo-tree"] then
+      vim.api.nvim_del_augroup_by_name "neotree_start"
+    else
+      local stats = vim.loop.fs_stat(vim.api.nvim_buf_get_name(0))
+      if stats and stats.type == "directory" then
+        vim.api.nvim_del_augroup_by_name "neotree_start"
+        require "neo-tree"
+      end
+    end
+  end,
+})
```

Comment:
Review with octo.nvim
```suggestion
autocmd("BufEnter", {
  desc = "Open Neo-Tree on startup with directory",
  group = augroup("neotree_start", { clear = true }),
  callback = function()
    if package.loaded["neo-tree"] then
      vim.api.nvim_del_augroup_by_name "neotree_start"
    else
      local stats = vim.loop.fs_stat(vim.api.nvim_buf_get_name(0))
      if stats and stats.type == "directory" then
        vim.api.nvim_del_augroup_by_name "neotree_start"
        require "neo-tree"
      end
    end
  end,
})
```